microsoft/ReportingServicesTools
# Initial Commit

Adding four functions to help automate deploying SSRS projects based on the deployment setting from the SSRS project file.

Fixes None.

Changes proposed in this pull request:
 - Creates two new 'helper' function, and two new functions for deployment
 - Creates a new Get-RsDeploymentConfig function for reading the SSRS project file
 - Creates a new Deploy-RsProject which deploys the SSRS project files based on the setting of the SSRS project file (which you can easily pipe in from the new Get-RsDeploymentConfig function)

How to test this code:
 - Take an SSRS 2019 project and deploy it to a Power BI Report Server
 - See examples in code.

Has been tested on (remove any that don't apply):
 - PowerShell 5.1
 - Windows 10
 - Power BI Report Server
